lib/common/tests/output/Makefile.am

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2021-2022 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU General Public License version 2
+# or later (GPLv2+) WITHOUT ANY WARRANTY.
+#
+
+AM_CPPFLAGS = -I$(top_srcdir)/include		\
+	      -I$(top_builddir)/include		\
+	      -I$(top_srcdir)/lib/common
+LDADD = $(top_builddir)/lib/common/libcrmcommon_test.la \
+		-lcmocka
+AM_CFLAGS = -DPCMK__UNIT_TESTING
+AM_LDFLAGS = $(LDFLAGS_WRAP)
+
+include $(top_srcdir)/mk/tap.mk
```

Comment:
Not for this PR, but would it be worthwhile to put all the common stuff into a mk/unittest.mk?

Reply:
Yeah, absolutely.  I think it's all the same now.